serverlessworkflow/modules/ROOT/pages/cloud/build-workflow-image-with-quarkus-cli.adoc

```diff
@@ -1,4 +1,15 @@
-= Building Serverless Workflow Images using Quarkus CLI
+Building Serverless Workflow Images using Quarkus CLI
+=====================================================
+v1.0, 06-03-2022
```

Comment:
What does this version signify?

Reply:
SMEs are using this version in their guides. 
@ricardozanini Could you please confirm what v1.0 signify?

Reply:
Done, removed version.